File: models/CandidateEducation.js
const Sequelize = require('sequelize');
const db = require("../config/db");
const User = require("./User");
const EducationField = require("./EducationField");
const EducationLevel = require("./EducationLevel");

const CandidateEducation = db.define("CandidateEducation", 
    {
        UserId: {
            type: Sequelize.INTEGER,
            allowNull: false
        },
        EducationLevelId: {
            type: Sequelize.INTEGER,
            allowNull: false
        },
        FieldId: {
            type: Sequelize.INTEGER,
            allowNull: false
        },
        From: {
            type: Sequelize.INTEGER,
            allowNull: false
        },
        To: {
            type: Sequelize.INTEGER
        }
    },
    {
        timestamps: false,
        freezeTableName: true
    }
);

module.exports = CandidateEducation;

File: API/Models/CandidateEducation.cs
using System;
using System.Collections.Generic;

namespace JobberAPI.Models;

public partial class CandidateEducation
{
    public int Id { get; set; }

    public int UserId { get; set; }

    public int EducationLevelId { get; set; }

    public int FieldId { get; set; }

    public int From { get; set; }

    public int To { get; set; }

    public virtual EducationLevel EducationLevel { get; set; } = null!;

    public virtual EducationField Field { get; set; } = null!;

    public virtual User User { get; set; } = null!;
}
